Code/Framework/AzToolsFramework/AzToolsFramework/ViewportUi/ViewportUiDisplay.cpp

```diff
@@ -450,6 +460,12 @@ namespace AzToolsFramework::ViewportUi::Internal
             region -= QRect(
                 QPoint(m_uiOverlay.rect().left() + HighlightBorderSize, m_uiOverlay.rect().top() + ViewportUiTopBorderSize),
                 QPoint(m_uiOverlay.rect().right() - HighlightBorderSize, m_uiOverlay.rect().bottom() - HighlightBorderSize));
+
+            if (m_viewportBorderText.width() != m_renderOverlay->width())
```

Comment:
update: Again a guide comment here for why we're doing this would be super helpful otherwise it's hard to know why it's required and what it's going to do

Reply:
Great comment update thanks 👍